checks.d/mongo.py

```diff
@@ -326,3 +391,29 @@ def check(self, instance):
             if m in self.RATES:
                 m = self.normalize(m.lower(), 'mongodb') + "ps"
                 self.rate(m, value, tags=tags)
+
+        stat_metrics = filter(lambda x: x.startswith('stats.'), self.METRICS)
+        for s in dbstats:
```

Comment:
`for s, value in dbstats.iteritems():`
`iteritems()` is a generator that yields 2-tupes representing the key and the value. This way s is the key and value is the value